src/main/java/duke/Parser.java

```diff
@@ -112,7 +118,7 @@ public static boolean isNumeric(String strNum) {
             return false;
         }
         try {
```

Comment:
I think it would be good to add a spacing right above the try block to separate the code logic between the two blocks.